algorithms/sorting.md

```diff
@@ -8,6 +8,15 @@
 - [Recursion](/recursion/recursion.md)
 - [Searching](/algorithms/searching.md) (Particularly Binary Search)
 
+### Motivation
+
+Sorting is important in programming for the same reason it is important in everyday life. It is easier and faster to locate items in a sorted list than unsorted. Sorting algorithms can be used in a program to sort an array for later searching or writing out to an ordered file or report.
+The point here isn't to remember the sorting algorithms which you can write on your job or in an interview, but, to understand what an algorithm actually is. We already have different libraries for this purpose but still, these algorithms serve as a good case study on algorithm analysis/design.
```

Comment:
can you remove this sentence?  I don't think it adds to the section.  "We already have different libraries for this purpose but still, these algorithms serve as a good case study on algorithm analysis/design."